packages/react-forms-dom/src/components/radio-group.tsx

```diff
@@ -45,9 +45,9 @@ export class RadioGroup extends BaseDomField<RadioGroupProps, BaseFieldState> {
         };
     }
 
-    protected get RawDefaultValue(): React.ReactText {
-        if (this.props.defaultValue != null) {
-            return this.props.defaultValue;
+    protected GetRawDefaultValue(props: RadioGroupProps): React.ReactText {
+        if (props.defaultValue != null) {
+            return props.defaultValue;
         }
 
```

Comment:
I'd rather keep it. And for inconsistencies, I think we'll have to create a proper TSLint rule for this sometime later.